ksql-engine/src/main/java/io/confluent/ksql/util/PersistentQueryMetadata.java

```diff
@@ -18,41 +18,48 @@
 import com.google.common.collect.ImmutableSet;
 import io.confluent.ksql.metastore.KsqlTopic;
 import io.confluent.ksql.metastore.StructuredDataSource;
-import io.confluent.ksql.planner.plan.OutputNode;
 import io.confluent.ksql.query.QueryId;
 import io.confluent.ksql.serde.DataSource;
 import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
 import java.util.function.Consumer;
+import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.streams.KafkaStreams;
 import org.apache.kafka.streams.Topology;
 
+/**
+ * Metadata of a persistent query, e.g. {@code SELECT CREATE STREAM FOO AS SELECT * FROM BAR;}.
```

Comment:
```suggestion
 * Metadata of a persistent query, e.g. {@code CREATE STREAM FOO AS SELECT * FROM BAR;}.
```

Reply:
and again... oops! Must be working too hard.